src/main/java/io/ebean/dbmigration/ddlgeneration/platform/BaseTableDdl.java

```diff
@@ -970,17 +993,20 @@ protected void addUniqueConstraint(DdlWrite writer, AlterColumn alter, String uq
 
 
   protected void alterTableDropColumn(DdlBuffer buffer, String tableName, String columnName) throws IOException {
-
-    buffer.append("alter table ").append(tableName).append(" drop column ").append(columnName)
-      .endOfStatement();
+    platformDdl.alterTableDropColumn(buffer, tableName, columnName);
   }
 
   protected void alterTableAddColumn(DdlBuffer buffer, String tableName, Column column, boolean onHistoryTable) throws IOException {
     DdlMigrationHelp help = new DdlMigrationHelp(tableName, column);    
-    help.writeBefore(buffer);
+    if (!onHistoryTable) {
```

Comment:
I think, the DbMigration should be applied only on main table.

Reply:
Yes, null and check constraints etc should not be on a history table per say.  So yes, migrations should only apply to the main table.